docs/BOOT_LOADER_SPECIFICATION.md

```diff
@@ -23,7 +23,7 @@ Of course, without this specification things already work mostly fine. But here'
 
 ## Why not simply rely on the EFI boot menu logic?
 
-The EFI specification provides a boot options logic that can offer similar functionality. Here's why we think that it is not enough for our uses:
+EFI is not ubiquitous, especially not in embedded or more dependable systems. Though, if you have an EFI system, it provides a boot options logic that can offer similar functionality. Here's why we think that it is not enough for our uses:
```

Comment:
hmm, the "more dependable" thing sounds political, I'd prefer if we'd drop that. this should not be the place to trash UEFI...